bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml

```diff
@@ -38,7 +38,7 @@ snowbridge-core = { path = "../../primitives/core", default-features = false }
 snowbridge-ethereum = { path = "../../primitives/ethereum", default-features = false }
 primitives = { package = "snowbridge-beacon-primitives", path = "../../primitives/beacon", default-features = false }
 static_assertions = { version = "1.1.0", default-features = false }
-bp-runtime = { path = "../../../../../bridges/primitives/runtime", default-features = false }
+bp-runtime = { path = "../../../../primitives/runtime", default-features = false }
```

Comment:
(this change won't beak anything in Snowbridge repo @claravanstaden ?)

Reply:
(also see similar changes below)

Reply:
It shouldn't. In the end it's the same path. Just optimized.

Reply:
I agree, but just wanted to double check :)